assignment-2/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Generally, it is a good idea to ignore also the `.vscode` and `.idea` folders (to not have conflicts with other people due to IDE settings).